packages/dso-toolkit/src/components/banner/banner.scss

```diff
@@ -68,33 +99,64 @@ dso-banner {
 
   .row > div::before {
     content: "";
+
     display: inline-block;
     left: units.$u2;
     position: absolute;
     top: 0;
   }
 
+  @mixin withIcon($variant) {
```

Comment:
- Volgens mij schrijven wij onze mixins altijd kebab-case
- Ik vind dat er erg veel basis styling wordt gegenerered, kan je uitzoeken of je deze styling kan centraliseren? Alleen `di.base()` maakt van `$variant` gebruik.